src/conditional-flow/multiple-conditions.js

```diff
@@ -1,19 +1,25 @@
 // This function should return true if num is greater
 // than or equal to lower AND less than or equal to upper.
 // Implement this with a single condition.
-function isInRange (num, lower, upper) {
+function isInRange(num, lower, upper) {
 
   // TODO: write code in this function body to pass the tests
-
+  if (num >= lower && num <= upper) {
```

Comment:
This can be simplified. `num >= lower && num <= upper` creates a boolean value representing if the num is greater than or equal to lower AND is less than or equal to upper. So we can simplify this function by just returning that value, like so:
```js

return num >= lower && num <= upper
```